src/Nethermind/Nethermind.Core.Test/Blockchain/TestBlockchain.cs

```diff
@@ -116,6 +117,13 @@ protected virtual async Task<TestBlockchain> Build(ISpecProvider? specProvider =
         DbProvider = await CreateDbProvider();
         TrieStore = new TrieStore(StateDb, LogManager);
         State = new WorldState(TrieStore, DbProvider.CodeDb, LogManager);
+
+        // Eip4788 precompile state account
+        if (specProvider?.GenesisSpec?.IsEip4788Enabled ?? false)
```

Comment:
Why is this needed? Shouldn't BlockProcessor create this for you?

Reply:
it's done for the processing of the genesis block, since the way our tests are hooked, without that line it expects a the state root of a state without the precompile account, which results in test failure in some test

Reply:
instead `?? false` use `== true` - way more readable.